### PR TITLE
New Interdyne Shipping Container - with fixes to old sprites

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

Adds 1 (2) new sprite to the Containers list to include also a default icon so it's no longer a checkerbox for the default. Mostly doing this for a mapping project and needed Interdyne added in, but hey why not bring it here and fix some of the blemishes as well. 

I also alphabetized them in the dmi and code block because OCD

<details>
<summary>Screenshots</summary>

image

image

image
</details>

## Why It's Good For The Game

Variety! 

## Changelog

:cl:Zergspower
imageadd: Containers - New Interdyne Container with new mapping default icon
fix: Containers - unified the apperance of each one, fixed blemishes and stray pixels
/:cl:

### CODE REVIEW

**Comment by Zergspower** [comment]
> You currently have a negative Fix/Feature pull request delta of -7. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

you mislabel 1 PR and you go so far in debt...

File: code/game/objects/structures/containers.dm [hunk 1]
```diff
@@ -14,44 +14,47 @@
...
+/obj/structure/shipping_container/conarex
+	name = "\improper Conarex Aeronautics shipping container"
+	desc = "A standard-measure shipping container for bulk transport of goods. This one is from Conarex Aeronautics, and is probably carrying spacecraft parts (or a bribery scandal) as a result."
+	icon_state = "conarex"
 
 /obj/structure/shipping_container/deforest
 	name = "\improper DeForest Medical Corp. shipping container"
 	desc = "A standard-measure shipping container for bulk transport of goods. This one is from DeForest, and so is probably carrying medical supplies."
 	icon_state = "deforest"
+/obj/structure/shipping_container/kahraman
```
Comment (JohnFulpWillard) [inline code/game/objects/structures/containers.dm:26]:
```suggestion

/obj/structure/shipping_container/kahraman
```

File: code/game/objects/structures/containers.dm [hunk 2]
```diff
@@ -14,44 +14,47 @@
 
 	AddComponent(/datum/component/seethrough, SEE_THROUGH_MAP_SHIPPING_CONTAINER)
 
-/obj/structure/shipping_container/nanotrasen
-	name = "\improper Nanotrasen shipping container"
-	desc = "A standard-measure shipping container for bulk transport of goods. This one prominently features Nanotrasen's logo, and so presumably could be carrying anything."
-	icon_state = "nanotrasen"
+/obj/structure/shipping_container/conarex
+	name = "\improper Conarex Aeronautics shipping container"
```
Comment (JohnFulpWillard) [inline code/game/objects/structures/containers.dm:18]: Who is Conarex Aeronautics? I thought it was a syndicate faction but I don't see them mentioned in https://github.com/tgstation/common_core/blob/master/Governments%20and%20Organisations/Corporations/The%20Syndicate/The%20Syndicate.md

Not anything serious I just wanna know

**Comment by JohnFulpWillard** [comment]
Updated your changelog, replaced add to imageadd

File: code/game/objects/structures/containers.dm (hunk 2 above)
Comment (Zergspower) [reply code/game/objects/structures/containers.dm:18]: lol man idk - that was the default in there so i just left it - maybe some distant lore? If you can think of any i wouldnt mind adding or editing

**Pull request merged**